server/src/main/java/org/apache/iotdb/db/query/dataset/groupby/GroupByWithoutValueFilterDataSet.java

```diff
@@ -51,10 +51,10 @@
 
 public class GroupByWithoutValueFilterDataSet extends GroupByEngineDataSet {
 
-  private static final Logger logger =
+  protected static final Logger logger =
```

Comment:
You should construct a new Logger using the Subclass in itself.

Reply:
done.